spotless-formatter-eclipse-jdt-configurations.xml

```diff
@@ -38,7 +38,6 @@
   <setting id="org.eclipse.jdt.core.formatter.parentheses_positions_in_if_while_statement" value="separate_lines_if_wrapped"/>
   <setting id="org.eclipse.jdt.core.formatter.alignment_for_arguments_in_enum_constant" value="48"/>
   <setting id="org.eclipse.jdt.core.formatter.insert_space_after_opening_angle_bracket_in_parameterized_type_reference" value="do not insert"/>
-  <setting id="org.eclipse.jdt.core.compiler.release" value="enabled"/>
```

Comment:
I  believe that the compiler options in the formatter file for spotless are unused, so these are removed